clang/test/CXX/drs/cwg25xx.cpp

```diff
@@ -139,7 +139,7 @@ struct D3 : B {
 #endif
 
 #if __cplusplus >= 202302L
-namespace cwg2561 { // cwg2561: no tentatively ready 2024-03-18
+namespace cwg2561 { // cwg2561: no ready 2024-06-25
```

Comment:
You should keep 2024-03-18 date, because there are no new resolution entries there.

Reply:
It doesn't really make sense to put a date for something Clang doesn't implement yet right? So I just removed the date. This one is now status=DR on https://raw.githubusercontent.com/cplusplus/CWG/gh-pages/issues/cwg_index.html so there isn't supposed to be a date anyway

Reply:
We need a date to understand what are we testing against that we reach the conclusion that we don't implement the DR

Reply:
That makes sense, thanks for explaining. For cwg2561/cwg2883 it shouldn't need a date now because they are in the standard draft: https://eel.is/c++draft/expr.prim.lambda.closure#9.sentence-1 https://eel.is/c++draft/basic.def.odr#10.2.3

Reply:
Hmm, https://cplusplus.github.io/CWG/issues/2561.html at the moment of writing says
> Section: 7.5.5.2  [[expr.prim.lambda.closure](https://wg21.link/expr.prim.lambda.closure)]     Status: ready     Submitter: Barry Revzin     Date: 2022-02-14     Liaison: EWG

while in cplusplus/CWG repo it's already DR

Reply:
It looks like the github pages hasn't been updated for a month: https://github.com/cplusplus/CWG/deployments/github-pages but make_cxx_dr_status pulls from raw.githubusercontent.com which is always the latest. I may have been reading some stale info if that is the case

Reply:
I sent Jens an email. Hopefully this will be fixed, or at least we'll get an explanation what the sync story is between pages and the repo.

Reply:
> I may have been reading some stale info if that is the case

Not just you :)
I'll also recheck your changes based on the repo instead of GH pages.

Reply:
Jens confirmed that GH pages are not supposed to deviate from the repo, and he's going to fix it.